### PR TITLE
Collect member name

### PR DESCRIPTION
**Pull request by bourrel**
https://github.com/TryGhost/Ghost/issues/12029

Add the ability to get member names from `data-member-forms`.

--------

- [x] There's a clear use-case for this code change
- [x] Commit message has a short title & references relevant issues
- [x] The build will pass (run `yarn test` and `yarn lint`)

### CODE REVIEW

**Comment by matthanley** [comment]
cc @rishabhgrg as similar may be required in Portal for backwards compat

File: core/server/public/members.js [hunk 1]
```diff
@@ -8,8 +8,10 @@ Array.prototype.forEach.call(document.querySelectorAll('form[data-members-form]'
             errorEl.innerText = '';
         }
         form.classList.remove('success', 'invalid', 'error');
-        var input = event.target.querySelector('input[data-members-email]');
-        var email = input.value;
+        var email_input = event.target.querySelector('input[data-members-email]');
```
Comment (allouis) [inline core/server/public/members.js:11]: We generally use camelCase rather than snake_case - would you be able to update? :relaxed:

**Review by allouis — CHANGES_REQUESTED** [review]
One comment - otherwise LGTM :tada:

**Comment by allouis** [comment]
Hey @bourrel this looks awesome!! Thanks for you contribution! I've tested this locally and it's working :ok_hand:

There's just one thing I missed, would you be able to run `grunt uglify` which will update the minified version of this file, and commit that - after that i'll merge this to master and it'll be in our next release :tada:

**Comment by bourrel** [comment]
Thanks, @allouis !

I've updated the uglified version, I hope this will be good for merging :)

**Comment by allouis** [comment]
:woman_dancing: And done, thanks for your work!

**Pull request merged**